src/groups/mwc/mwcu/mwcu_printutil.cpp

```diff
@@ -306,4 +308,4 @@ bsl::ostream& prettyTimeInterval(bsl::ostream&      stream,
 }  // close PrintUtil namespace
 
 }  // close package namespace
-}  // close enterprise namespace
+}  // close enterprise namespace
```

Comment:
[Text files should end with newlines](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline), but more importantly this looks like a stray change that wasn't intentional.  It's a good idea generally to double check these things (`git show`, or use `git add -p` when staging), and that your patches are all and only what you intend to submit.  No big issue